tests/security/test_check_secret_expiration.py

```
@@ -0,0 +1,474 @@
+        medium_risk = get_secrets_by_risk_level(RiskLevel.MEDIUM)
+
+        assert len(medium_risk) == 1
+        assert medium_risk[0].name == "CURRENT_SECRET"
+
+
+class TestReportGeneration:
+    """Test report generation functions"""
+
+    def create_sample_secrets(self) -> list[Secret]:
+        """Create sample secrets for report testing"""
+        now = datetime.now()
```

Ruff (0.14.4)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)